cumulusci/core/config/ScratchOrgConfig.py

```diff
@@ -141,9 +141,13 @@ def email_address(self):
     def days(self):
         return self.config.setdefault("days", 1)
 
+    @property
+    def alive(self):
+        return self.date_created and not self.expired
+
     @property
     def expired(self):
-        return self.expires and self.expires < datetime.datetime.now()
+        return bool(self.expires) and self.expires < datetime.datetime.now()
```

Comment:
Is there an actual change in behavior here or is it just to be clearer about what we're expecting to get?

Reply:
Previously this method could return None/True/False.

True = Expired
False = Active
None = Never created so not expired

Not it returns True or False.

True = Expired
False = Active or Never created so not expired

Trinary is kind of unusual and confusing.

Reply:
Oh yeah, I see what you mean now.